compiler_opt/rl/local_data_collector_test.py

```diff
@@ -21,16 +21,14 @@
 from typing import List, Tuple
 
 import tensorflow as tf
-from tf_agents.system import system_multiprocessing as multiprocessing
-
 # This is https://github.com/google/pytype/issues/764
 from google.protobuf import text_format  # pytype: disable=pyi-error
-from compiler_opt.distributed.local.local_worker_manager import LocalWorkerPoolManager
-from compiler_opt.rl import compilation_runner
-from compiler_opt.rl import corpus
-from compiler_opt.rl import data_collector
-from compiler_opt.rl import local_data_collector
-from compiler_opt.rl import policy_saver
+from tf_agents.system import system_multiprocessing as multiprocessing
+
+from compiler_opt.distributed.local.local_worker_manager import \
+    LocalWorkerPoolManager
+from compiler_opt.rl import (compilation_runner, corpus, data_collector,
```

Comment:
I think we want these one per line, for some stylistic reason that I now realize isn't supported by pylint of yapf (or we didn't figure out how to express it)

Reply:
oh sorry, this is another discrepancy between my vim on-save autoformatter and yapf. will fix